packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -209,6 +209,25 @@ export type PgEntity =
   | PgExtension
   | PgIndex;
 
+const fakeEnumIdentifier = (
+  namespaceName: string,
+  name: string,
+  constraint?: PgConstraint,
+  list = false
+) => {
+  const constraintIdent = constraint
+    ? constraint.type === "p"
+      ? ""
+      : `_${constraint.name}`
+    : "";
+  const baseEnumIdentifier = `FAKE_ENUM_${namespaceName}_${name}${constraintIdent}`;
```

Comment:
`constraint` should either be required (in which case you have to pass the primary key constraint if you don't know), or it should just be a string (in which case you can pass "" with a comment indicating it's referencing the primary key)

```suggestion
  identifierExtra = "",
  list = false
) => {
  const baseEnumIdentifier = `FAKE_ENUM_${namespaceName}_${name}${identifierExtra}`;
```